device/virtual/x86_64-kvm_x86_64-r0/msft_multi_asic_vs/topology.sh

```diff
@@ -8,10 +8,10 @@ NUM_INTERFACES_PER_ASIC=32
 
 start () {
     # Move external links into assigned frontend namespaces
-    # eth0  - eth15: asic2 
-    # eth16 - eth31: asic3 
-    # eth32 - eth47: asic4
-    # eth48 - eth63: asic5
+    # eth1  - eth16: asic0
+    # eth17 - eth32: asic1 
+    # eth33 - eth48: asic2
+    # eth49 - eth64: asic3
     for ASIC in `seq $FIRST_FRONTEND_ASIC $LAST_FRONTEND_ASIC`; do
         for NUM in `seq 1 16`; do
             ORIG="eth$((16 * $ASIC + $NUM))"
```

Comment:
Looks good, had a question of whether we need to really rename the interface to eth1-eth16 in all the namespaces ? We could have moved the interface as is ? Was there some issue with libvssai if we did that ?

Reply:
yes, vs has a lanemap.ini file which uses the interface names eth1 , eth2.. and maps each interface to lanes as defined in port_config.ini.